Source/WebCore/crypto/SubtleCrypto.cpp

```diff
@@ -96,7 +98,6 @@ static ExceptionOr<std::unique_ptr<CryptoAlgorithmParameters>> normalizeCryptoAl
     if (std::holds_alternative<String>(algorithmIdentifier)) {
         auto newParams = Strong<JSObject>(vm, constructEmptyObject(&state));
         newParams->putDirect(vm, Identifier::fromString(vm, "name"_s), jsString(vm, std::get<String>(algorithmIdentifier)));
-        
```

Comment:
not needed

Reply:
what is exactly what is not needed ?

Reply:
We usually do not remove lines except if changing the file.
There is no need to change SubtleCrypto.cpp in this patch.